spaces/farm/index.json

```
@@ -14,10 +14,15 @@
   "min": 1,
   "invalid": [],
   "strategies": [
+["erc20-balance-of", {
+      "address": "0x25550Cccbd68533Fa04bFD3e3AC4D09f9e00Fc50",
+      "symbol": "FARM_SP",
```

Can you change to this? It will be similar than on others spaces. 
```suggestion
      "symbol": "FARM (SP)",
```

---

ok done